appservice/src/siteFiles.ts

```diff
@@ -20,12 +19,20 @@ export interface ISiteFileMetadata {
     mime: string;
     name: string;
     path: string;
+    href: string;
 }
 
-export async function getFile(context: IActionContext, site: ParsedSite, filePath: string): Promise<ISiteFile> {
+/**
+ * @param path - Do not include leading slash. Include trailing slash if path represents a folder.
+ */
+export function createSiteFilesHref(site: ParsedSite, path: string): string {
+    return `${site.kuduUrl}/api/vfs/${path}?api-version=2022=03-01`
```

Comment:
I'm pretty sure that Kudu doesn't have an api-version. That was my mistake when migrating over Kudu calls (because I was thinking it was an ARM call)

Reply:
Removed it.